docs/README.md

```diff
@@ -0,0 +1,18 @@
+# Documentation of CredSweeper
+
+The directory is used for documentation of CredSweeper with using [sphinx](https://www.sphinx-doc.org/en/master/),
+
+## Workflow
+
+There is applied custom documentation, so auto-generation might fail. Please, use auto-generation as reference.
+With the command new sources might be updated (in /docs directory):
+
+```bash
+sphinx-apidoc --force --full --ext-autodoc ../credsweeper -o source/
+```
```

Comment:
What about add those command to github actions?
It can be a good test of document i think.

Reply:
The command does not generate correct .rst files for current config (custom main page and so on). 
Maybe in next PR...

Reply:
Okay. Thank you!